debian/rules

```diff
@@ -17,6 +17,10 @@ override_dh_auto_build:
 			-X $(DH_GOPKG)/ui.Build=$(BUILD_DATE)\
 		"
 
+override_dh_installinit:
```

Comment:
This is required to install multiple services. Doesn't change any other behaviour.